### ISSUE TITLE
[BUG] tezos-node does't find zcash params on MacOS

### ISSUE THREAD

**Issue by itkach** [serokell/tezos-packaging#385]
# Description
tezos-node fails to execute anything because it cannot find Zcash params

```
❯ brew install tezos-node-ithacanet
Warning: Calling bottle :unneeded is deprecated! There is no replacement.
Please report this issue to the serokell/tezos-packaging-rc tap (not Homebrew/brew or Homebrew/core):
  /opt/homebrew/Library/Taps/serokell/homebrew-tezos-packaging-rc/Formula/tezos-node-ithacanet.rb:9

==> Downloading file:///dev/null

Warning: Cannot verify integrity of 'fa6444b99c49cd66f605ad608ca5df7c1e8e4a20f71f1a26f246bdd0096f461b--null'.
No checksum was provided for this resource.
For your reference, the checksum is:
  sha256 "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
==> Installing tezos-node-ithacanet from serokell/tezos-packaging-rc
Warning: Calling bottle :unneeded is deprecated! There is no replacement.
Please report this issue to the serokell/tezos-packaging-rc tap (not Homebrew/brew or Homebrew/core):
  /opt/homebrew/Library/Taps/serokell/homebrew-tezos-packaging-rc/Formula/tezos-node-ithacanet.rb:9

Installing tezos-node-ithacanet serviceWarning: Calling bottle :unneeded is deprecated! There is no replacement.
Please report this issue to the serokell/tezos-packaging-rc tap (not Homebrew/brew or Homebrew/core):
  /opt/homebrew/Library/Taps/serokell/homebrew-tezos-packaging-rc/Formula/tezos-node-ithacanet.rb:9

==> tezos-node config init --data-dir/opt/homebrew/var/lib/tezos/node-ithacanet --network ithacanet
Last 15 lines from /Users/itkach/Library/Logs/Homebrew/tezos-node-ithacanet/post_install.01.tezos-node:

tezos-node
config
init
--data-dir/opt/homebrew/var/lib/tezos/node-ithacanet
--network
ithacanet

Failed to initialize Zcash parameters: cannot find Zcash params in any of:
- /Users/itkach/.local/share/zcash-params
- /Users/itkach/.zcash-params
- /Users/itkach/_opam/share/zcash-params
- /usr/local/share/zcash-params
- /usr/share/zcash-params
You may download them using https://raw.githubusercontent.com/zcash/zcash/master/zcutil/fetch-params.sh
Warning: The post-install step did not complete successfully
```

# To Reproduce

- `brew install tezos-node-ithacanet`
- observe tezos-node config init command fails

# Expected behavior

no error

# Environment

- OS: MacOS Monterey
- branch/revision: v12.0-rc2-1
- tezos-sapling-params installed
```
❯ brew install tezos-sapling-params
Warning: serokell/tezos-packaging-rc/tezos-sapling-params v8.2-3 is already installed and up-to-date.
To reinstall v8.2-3, run:
  brew reinstall tezos-sapling-params
```

# Additional context

**Comment by rvem** [comment]
Seems like it's impossible to workaround this nicely from `brew` side. I'll propose to add `$HOMEBREW_PREFIX/share/zcash-params` to the list of directories in which `zcash-params` are searched

**Comment by rvem** [comment]
I've submitted a patch to the upstream: https://gitlab.com/tezos/tezos/-/merge_requests/4609.
Hopefully, it'll be included in the next release

**Comment by rvem** [comment]
A fix was included in v13.0, so I'm closing this issue
